src/jit/objectalloc.cpp

```diff
@@ -321,6 +392,8 @@ bool ObjectAllocator::MorphAllocObjNodes()
 
                     const unsigned int stackLclNum = MorphAllocObjNodeIntoStackAlloc(asAllocObj, block, stmt);
                     m_HeapLocalToStackLocalMap.AddOrUpdate(lclNum, stackLclNum);
+                    MarkLclVarAsDefinitelyStackPointing(lclNum);
+                    MarkLclVarAsPossiblyStackPointing(lclNum);
```

Comment:
Can you add a note here and/or elsewhere that the possibly stack pointing set is kept as a superset of the definitely stack pointing set (and a local is in both it is definitely stack pointing).